### PR TITLE
Add metric to aws/cloudwatch-alarm massdriver_package_alarm

### PR DESCRIPTION
**Pull request by akasprzok**

### CODE REVIEW

File: aws/cloudwatch-alarm/main.tf [hunk 1]
```diff
@@ -25,4 +25,10 @@ resource "aws_cloudwatch_metric_alarm" "alarm" {
 resource "massdriver_package_alarm" "package_alarm" {
   display_name      = var.display_name
   cloud_resource_id = aws_cloudwatch_metric_alarm.alarm.arn
+  metric {
```
Comment (xpositivityx) [inline aws/cloudwatch-alarm/main.tf:28]: nice!

**Comment by mclacore** [comment]
Tested this change with `aws-vpc`:
- [] In-place upgrade
- [] In-place decommission w/o upgrade
- [] Fresh deployment

All tests passed

**Review by mclacore — APPROVED** [review]
LGTM

**Comment by mclacore** [comment]
closes: https://linear.app/massdriver/issue/ORC-319/massdriver-package-alarm-update-aws

**Pull request merged**